cmd/up/activate.go

```diff
@@ -423,7 +423,7 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context,
 				continue
 			case "TriggeredScaleUp":
 				failedSchedulingEvent = nil
-				oktetoLog.Spinner("Waiting for the cluster to scale up...")
+				oktetoLog.Spinner("Waiting for the context to scale up...")
```

Comment:
```suggestion
				oktetoLog.Spinner("Waiting for the cluster to scale up...")
```
I thing in this case is clearer talk about cluster